### ISSUE TITLE
[Question] Is OpenDoas support SELinux?

### ISSUE THREAD

**Issue by amalgame21** [Duncaen/OpenDoas#118]
I saw in arch wiki that `sudo` has to be rebuild with the `--with-selinux` flag if SELinux is installed, but I cannot see similar thing available on OpenDoas.

**Comment by TheDcoder** [comment]
I don't think it does, and for what it's worth, Arch Linux doesn't support SELinux either.
